src/common/common-types.ts

```diff
@@ -58,6 +61,8 @@ export interface Output {
      */
     readonly neverReason?: string | null;
     readonly label: string;
+    readonly kind: OutputKind;
+    readonly hasHandle: boolean;
```

Comment:
The context pane now needs to check whether outputs have handles to connect to. It needs to be added to the `getFirstPossibleOutput`, I think. Please add that check.

Reply:
Oh yeah, forgot about that.